package com.ex.exceptions;

public class UnderstandingExceptions {

	public static void main(String[] args) {
		
		int[] arr = new int[4];
		
		//try something
		try {
			arr[0] = 1;
			arr[4] = 5;
			
			System.out.println(arr[4]);
		} catch (ArrayIndexOutOfBoundsException e) { //do this if caught array bounds excp.
			System.out.println("caught out of bounds");
			// doesnt print this: e.printStackTrace();
		}
		
		System.out.println("passed");
	}
}
